consensus/types/src/beacon_state/tests.rs

```diff
@@ -68,31 +90,23 @@ fn test_beacon_proposer_index<T: EthSpec>() {
 
 #[test]
 fn beacon_proposer_index() {
-    test_beacon_proposer_index::<MinimalEthSpec>();
+    test_beacon_proposer_index::<eth_spec::MinimalEthSpec>();
```

Comment:
```suggestion
    test_beacon_proposer_index::<MinimalEthSpec>();
```

Can drop the `eth_spec::` throughout